Content.Shared/Implants/Components/ImplanterComponent.cs

```diff
@@ -70,6 +70,12 @@ public sealed partial class ImplanterComponent : Component
     [DataField]
     public (string, string) ImplantData;
 
+    /// <summary>
+    /// If the <see cref="Implant"/> is already implanted, should it be allowed to be implanted again?
```

Comment:
This comment isnt really accurate to what it does? Should probably be more clear

Reply:
I don't really know how to explain it better